### PR TITLE
Multi auth

### PR DESCRIPTION
**Pull request by chitalian**

### CODE REVIEW

File: worker/src/lib/RequestWrapper.ts [hunk 1]
```diff
@@ -18,13 +18,55 @@ export class RequestWrapper {
...
+    if (!authorization) {
+      return request.headers;
+    }
+    if (
+      !authorization.includes(",") ||
+      !authorization.includes("helicone-sk-")
+    ) {
+      return request.headers;
+    }
+    console.log("MUTATING AUTHORIZATION HEADER", authorization);
```
Comment (chitalian) [inline worker/src/lib/RequestWrapper.ts:41]: remove all console logs

File: worker/src/lib/RequestWrapper.ts [hunk 2]
```diff
@@ -18,13 +18,55 @@ export class RequestWrapper {
...
+      return request.headers;
+    }
+    if (
+      !authorization.includes(",") ||
+      !authorization.includes("helicone-sk-")
+    ) {
+      return request.headers;
+    }
+    
+
```
Comment (chitalian) [inline worker/src/lib/RequestWrapper.ts:42]: remove weird spacing

**Review by colegottdank — APPROVED** [review]
Its ight

**Comment by wwzeng1** [comment]
sweet

**Pull request merged**